docs/Remove-VSTeamAccessControlEntry.md

```diff
@@ -1,203 +0,0 @@
-

```

Comment:
I did not, the `Build-Module.ps1` -buildHelp did :)
It looks like the documentation was commited wrongly directly to /docs/ instead of /.docs/ which is the source for /docs/.

This is actually confusing and i wonder if we can improve this somehow using the build pipeline? (only committing /.docs/)?

Reply:
This is not /.docs/, it's /docs/